recipe/meta.yaml

```
@@ -1,63 +1,298 @@
+    - patches/0010-Ensure-sql_main-uses-protobuf-shared-library-correct.patch
+    - patches/0011-Include-missing-header-for-std-isdigit.patch
+    - patches/0012-Print-number-of-lines-in-the-file-generated-by-creat.patch
+    - patches/0013-Check-for-ARMv8-crc32-intrinsic-support-on-osx-arm64.patch
+    - patches/0014-Revert-Bug-30981914-COMPILE-WITH-FASTER-TLS-MODEL.patch
+    - patches/0015-Zlib-module-use-macro-instead-of-function.patch
+    - patches/0016-Make-get_relative_path-binary-patching-friendly.patch
+    ## Avoid crash on ppc64le, aarch64
+    - patches/0017-Revert-BUG-34849343-Aligned_atomic-not-working-as-in.patch
+    - patches/0018-Make-libfido-checks-conditional.patch
+    - patches/0019-Don-t-include-headers-from-Homebrew.patch
+    - patches/0020-Use-config-mode-search-for-protobuf-first.patch
+    - patches/0021-Simplify-protobuf-deps-using-cmake-alias.patch
+    - patches/0022-Set-PROTOBUF_INCLUDE_DIR-only-if-not-set.patch
+    - patches/0023-Allow-setting-WITH_SASL-none.patch
+    - patches/0024-Dump-std-out-err-if-abi-check-fails.patch
+    - patches/0025-Remove-unsupported-flag-from-macos-linker.patch
 
 build:
-  number: 2
-  skip: True # [win]
+  number: 0
 
 requirements:
   build:
-    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - patch    # [not win]
-    - m2-patch # [win]
-    - patchelf # [linux]
     - cmake
     - ninja
-    - bison    # [unix]
+    - bison                    # [unix]
+    - diffutils                # [unix]
+    - msys2-bison              # [win]
     - pkg-config
-
+    - icu                      # [target_platform == 'osx-arm64']
+    - libprotobuf              # [target_platform == 'osx-arm64']
+    - openssl                  # [target_platform == 'osx-arm64']
+    - zlib                     # [target_platform == 'osx-arm64']
+    - zstd                     # [target_platform == 'osx-arm64']
+    - git                      # [unix]
+    - patch                    # [unix]
+    - msys2-patch              # [win]
   host:
     - icu {{ icu }}
-    - libcurl 8.7.1
-    - libedit 3.1.20230828 # [unix]
-    - libprotobuf 5.29.3
-    - libabseil 20250127.0
-    - lz4-c 1.9.4
+    - libabseil {{ libabseil }}
+    - libedit {{ libedit }}  # [unix]
+    - libprotobuf {{ libprotobuf }}
+    - lz4-c {{ lz4_c }}
+    - openldap {{ openldap }}  # [unix]
     - openssl {{ openssl }}
     - zlib {{ zlib }}
     - zstd {{ zstd }}
-test:
-  commands:
-    - mysql_config --version
-    - my_print_defaults --version
-    - mysqld --help --verbose
-    - command -v mysql.server
+
+
+outputs:
+  - name: mysql-server
+    script: scripts/install-server.sh   # [unix]
+    script: scripts/install-server.bat  # [win]
+    build:
+      # Used only for lber.h, no library needed at runtime
+      ignore_run_exports:
+        - openldap
+    requirements:
+      build:
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja-base
+      host:
+        - icu
```

if deps in conda_build_config add {{ pkg_form_cbc }}
for example icu: https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L338
```suggestion
        - icu {{ icu }}
```